src/api/reducers/entities.js

```diff
@@ -7,6 +7,7 @@ export const entities = (
     orgs: {},
     repos: {},
     users: {},
+    gqlRepos: {},
```

Comment:
Why are we separating it into it's own entity instead of just using `repos`? I'm assuming this is because other parts of the app still reference the original `repos` object (and if that's the case -> not a big deal for now)

Reply:
`repos` here is from our new store architecture. It holds v3 repositories found in the Search screen, or starred by a user for example.

I needed to separate repositories retrieved via GraphQL , from the one retrieved from the Rest API, because they don't share the same structure / field names.

---

If you get back to #761, you'll see that I've mentioned adapting GraphQL entities structure to match the ones from Rest. 

But now that I've experimented a bit with GraphQL, I think that we should be doing exactly the opposite: Adapt REST entities to the GraphQL structure in our store. We will do that once we migrated everything to the new API, because it will require updates in most of our components (avatar_url => avatarUrl).

---

Furthermore, since we're retrieving **exactly** what we need from GraphQL, I think that we should be storing the response structure as-is in the store. I'll be updating this PR accordingly to reflect that (and loose the ugly code in  gqlRepos schema)